packages/pieces/notion/package.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "@activepieces/piece-notion",
-  "version": "0.2.2"
+  "version": "0.3.2"
```

Comment:
Please bump the minor version as it is backward compatible

Reply:
Is it should be 0.4.2? I have already increased it to 0.3.2 from 0.2.2

Reply:
It should be 0.2.3